### PR TITLE
Let's Go Herping 模组翻译

### PR DESCRIPTION
**Pull request by SweetLemone**
- [x] 我已**仔细**阅读贡献指南 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)；
- [x] 我已确认英文原文（如 en_us.json）存在且完整，内容与中文对应；
- [x] 我已阅读并同意按 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh) 协议发布我的作品；
- [x] 刷新 PR 的标签/状态，有需要再点击；

### CODE REVIEW

File: projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "item.herping.fat_tailed_gecko_spawn_egg": "非洲肥尾壁虎卵",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json:2]:
```suggestion
  "item.herping.fat_tailed_gecko_spawn_egg": "非洲肥尾壁虎刷怪蛋",
```
mc术语，请全文修改

File: projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "item.herping.amazon_milk_frog_tadpole_egg": "亚马逊牛奶蛙蝌蚪卵",
+  "item.herping.blue_nosed_chameleon_egg": "蓝鼻变色龙卵",
+  "item.herping.green_anole_spawn_egg": "绿安乐蜥卵",
+  "item.herping.tentacled_snake_spawn_egg": "吻突蝣蛇卵",
+
+  "item.herping.mealworm": "面包虫",
+  "item.herping.superworm": "大麦虫",
+  "item.herping.milk_frog_tadpole_bucket": "亚马逊牛奶蛙蝌蚪桶",
+
+  "itemGroup.herping": "找寻两栖动物吧!",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json:22]:
```suggestion
  "itemGroup.herping": "找寻两栖动物吧！",
```

**Review by ChuijkYahus — COMMENTED** [review]
物种名称待查，今晚看看（）

**Comment by SweetLemone** [comment]
之前是写的刷怪蛋，然后我一寻思，两栖动物，卵多好（）



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月2日(星期天) 晚上6:39
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] 提交Let&#39;s Go Herping模组翻译 (PR #3439)





 
@ChuijkYahus commented on this pull request.
 
物种名称待查，今晚看看（）

 
In projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json:
 &gt; @@ -0,0 +1,38 @@ +{ +  "item.herping.fat_tailed_gecko_spawn_egg": "非洲肥尾壁虎卵",  ⬇️ Suggested change -  "item.herping.fat_tailed_gecko_spawn_egg": "非洲肥尾壁虎卵", +  "item.herping.fat_tailed_gecko_spawn_egg": "非洲肥尾壁虎刷怪蛋",  
mc术语，请全文修改
 
 
In projects/1.18/assets/lets-go-herping/herping/lang/zh_cn.json:
 &gt; +  "item.herping.jacksons_chameleon_spawn_egg": "杰克森变色龙卵", +  "item.herping.emperor_newt_spawn_egg": "红瘰疣螈卵", +  "item.herping.fiji_banded_iguana_spawn_egg": "斐济带纹鬣蜥卵", +  "item.herping.crocodile_skink_spawn_egg": "红眼鹰蜥卵", +  "item.herping.bearded_dragon_spawn_egg": "鬃狮蜥卵", +  "item.herping.pancake_turtle_spawn_egg": "非洲软甲陆龟卵", +  "item.herping.amazon_milk_frog_tadpole_egg": "亚马逊牛奶蛙蝌蚪卵", +  "item.herping.blue_nosed_chameleon_egg": "蓝鼻变色龙卵", +  "item.herping.green_anole_spawn_egg": "绿安乐蜥卵", +  "item.herping.tentacled_snake_spawn_egg": "吻突蝣蛇卵", + +  "item.herping.mealworm": "面包虫", +  "item.herping.superworm": "大麦虫", +  "item.herping.milk_frog_tadpole_bucket": "亚马逊牛奶蛙蝌蚪桶", + +  "itemGroup.herping": "找寻两栖动物吧!",  ⬇️ Suggested change -  "itemGroup.herping": "找寻两栖动物吧!", +  "itemGroup.herping": "找寻两栖动物吧！",

**Comment by mamaruo** [comment]
标题不合规

**Comment by SweetLemone** [comment]
请问是哪里不合规



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月2日(星期天) 晚上7:21
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CFPAOrg/Minecraft-Mod-Language-Package] 提交Let&#39;s Go Herping模组翻译 (PR #3439)






标题不合规

—
直接回复此邮件，在GitHub上查看，或者取消订阅.
***@***.***与&gt;.

**Comment by mamaruo** [comment]
image

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md#github-pr

**Review by mamaruo — COMMENTED** [review]
语言文件不是最新版的，请补充。
<details><summary>Details</summary>
<p>

{
  "item.herping.fat_tailed_gecko_spawn_egg": "African Fat-Tailed Gecko Spawn Egg",
  "item.herping.african_spurred_tortoise_spawn_egg": "African Spurred Tortoise Spawn Egg",
  "item.herping.argentine_tegu_spawn_egg": "Argentine Tegu Spawn Egg",
  "item.herping.tiger_salamander_spawn_egg": "Tiger Salamander Spawn Egg",
  "item.herping.milk_frog_spawn_egg": "Milk Frog Spawn Egg",
  "item.herping.jacksons_chameleon_spawn_egg": "Jacksons Chameleon Spawn Egg",
  "item.herping.emperor_newt_spawn_egg": "Emperor Newt Spawn Egg",
  "item.herping.fiji_banded_iguana_spawn_egg": "Fiji Banded Iguana Spawn Egg",
  "item.herping.crocodile_skink_spawn_egg": "Red-Eyed Crocodile Skink Spawn Egg",
  "item.herping.bearded_dragon_spawn_egg": "Bearded Dragon Spawn Egg",
  "item.herping.pancake_turtle_spawn_egg": "African Pancake Tortoise Spawn Egg",
  "item.herping.amazon_milk_frog_tadpole_egg": "Amazon Milk Frog Tadpole Spawn Egg",
  "item.herping.blue_nosed_chameleon_egg": "Blue-Nosed Chameleon Spawn Egg",
  "item.herping.green_anole_spawn_egg": "Green Anole Spawn Egg",
  "item.herping.tentacled_snake_spawn_egg": "Tentacled Snake Spawn Egg",
  "item.herping.spiny_tailed_gecko_spawn_egg": "Northern Spiny-Tailed Gecko Spawn Egg",
  "item.herping.leaf_tailed_gecko_spawn_egg": "Leaf-Tailed Gecko Spawn Egg",

  "item.herping.green_anole_pot": "Green Anole Pot",
  "item.herping.crocodile_skink_pot": "Red-Eyed Crocodile Skink Pot",
  "item.herping.blue_nosed_chameleon_pot": "Blue-Nosed Chameleon Pot",

  "item.herping.spiny_tailed_gecko_bucket": "Bucket of Spiny-Tailed Gecko",
  "item.herping.fat_tailed_gecko_bucket": "Bucket of African Fat-Tailed Gecko",
  "item.herping.leaf_tailed_gecko_bucket": "Bucket of Leaf-Tailed Gecko",
  "item.herping.emperor_newt_bucket": "Bucket of Emperor Newt",

  "item.herping.mealworm": "Mealworm",
  "item.herping.superworm": "Superworm",
  "item.herping.milk_frog_tadpole_bucket": "Bucket of Amazon Milk Frog Tadpole",

  "itemGroup.herping": "Lets Go Herping!",

  "entity.herping.fat_tailed_gecko": "African Fat Tailed Gecko",
  "entity.herping.pancake_turtle": "African Pancake Tortoise",
  "entity.herping.african_spurred_tortoise": "African Spurred Tortoise",
  "entity.herping.milk_frog": "Amazon Milk Frog",
  "entity.herping.argentine_tegu": "Argentine Tegu",
  "entity.herping.bearded_dragon": "Bearded Dragon",
  "entity.herping.blue_nosed_chameleon": "Blue-Nosed Chameleon",
  "entity.herping.emperor_newt": "Emperor Newt",
  "entity.herping.fiji_banded_iguana": "Fiji Banded Iguana",
  "entity.herping.green_anole": "Green Anole",
  "entity.herping.jackons_chameleon": "Jackson's Chameleon",
  "entity.herping.leaf_tailed_gecko": "Leaf-Tailed Gecko",
  "entity.herping.crocodile_skink": "Red-Eyed Crocodile Skink",
  "entity.herping.spiny_tailed_gecko": "Northern Spiny-Tailed Gecko",
  "entity.herping.tentacled_snake": "Tentacled Snake",
  "entity.herping.tiger_salamander": "Tiger Salamander"
}

</p>
</details>

**Comment by ChuijkYahus** [comment]
找了一下其他的物种

模组英文|学名|链接|建议
-|-|-|-
African Fat-Tailed Gecko|*Hemitheconyx caudicinctus*|[Wikipedia-en](https://en.wikipedia.org/wiki/African_fat-tailed_gecko)<br>[iNaturalist](https://www.inaturalist.org/taxa/34422-Hemitheconyx-caudicinctus)<br>[百度百科](https://baike.baidu.com/item/%E8%82%A5%E5%B0%BE%E5%AE%88%E5%AE%AB/649353?fr=ge_ala)|肥尾守宫、肥尾壁虎
Blue-Nosed Chameleon|*Calumma beottgeri*|[Wikipedia-en](https://en.wikipedia.org/wiki/Calumma_boettgeri)<br>[iNaturalist](https://www.inaturalist.org/taxa/488307-Calumma-boettgeri)|
*|*Calumma linotum*|[Wikipedia-en](https://en.wikipedia.org/wiki/Calumma_linotum)<br>[iNaturalist](https://www.inaturalist.org/taxa/488308-Calumma-linotum)|

Leaf-tailed Gecko貌似是个很泛的称呼（）
[Jackson's Chameleon](https://www.iucnredlist.org/species/172531/1344462)和[Amazon Milk Frog](https://www.iucnredlist.org/species/55823/11373135)都没找到除维百来源外的中文学名

**Comment by SweetLemone** [comment]
更新啥啊，懒得陪你们玩了。能告诉我非洲肥尾壁虎，肥尾守宫，肥尾壁虎区别在哪？守宫和壁虎区别在哪？边境牧羊犬建议改名叫做边境牧羊狗？改成边境柯利犬我都能理解。至于苏卡达陆龟，我想我已经写清楚了，另外非洲刺龟就是它别名，哪里翻译错误我接受，不接受鸡蛋里面挑骨头。癞疙宝不是癞蛤蟆是吧？怎么，我翻译番薯是番薯，你建议我翻译番薯是红薯对吗？找不到中文学名会根据模组里面的动物形态判断应该是哪个动物吗？有没有可能牛奶蛙就是亚马孙/亚马逊牛奶蛙呢？有没有可能杰克森变色龙就是国内叫法呢？并且和模组里面的特征是对应的呢？你这土豆不应该叫洋芋应该叫马铃薯？生物方面没这么过分严苛，你想怎么叫就怎么叫，只要你不叫错它的拉丁文学名。只有拉丁文学名是死的，固定的，因为拉丁文是一门死语言。真受不了诶？翻译本身就随心的，心有猛虎，细嗅蔷薇也能翻译成心中有只老虎，闻了闻玫瑰，谁能指责你吗？这模组翻译本就我随心而为，各个翻译也都没出差错，指出问题我接受，我真接受，指出来错误，而不是你这番茄建议叫西红柿，你这樱桃建议叫车厘子，你这菠萝建议叫凤梨，你这狗建议叫犬，你这猪建议叫豚。你可以建议，我也可以不接受建议。说白了我翻译的时候也就只想着给自己用，后来想着分享出来，仅此而已。你们可以随便要求，随便建议，请便，我不干涉，也请自己动手修改。你说我物种搞错了，我绝对是道歉三连，对不起，我错了，请原谅。至于你们这种建议？算了吧。然后，苏卡达陆龟和饼干龟我都还能接受，因为它们确实一个是苏卡达龟，种，一个是饼干龟，种。

至于语言更新，额，模组更新我不知道，刚从外地回来，刚去CF查找了一下，1.18.2还是1.3，可能是作者没传CF，至于更新翻译的话，没兴趣啦，谁知道你们是要拉丁文音译名，英文直译名，还是中文学名，国内俗名呢？

最后不管是i18,还是i18n,我说诉说的自己为mod文件的lang里面加入汉化文件，每次启动游戏之后都得卸载I18n资源包才会汉化生效的事情，你们自己都不在乎，还在这种莫名其妙的事情上针尖挑刺？

**Comment by mamaruo** [comment]
> 至于语言更新，额，模组更新我不知道，刚从外地回来，刚去CF查找了一下，1.18.2还是1.3，可能是作者没传CF，至于更新翻译的话。

我眼拙看成 1.19 的文件了。抱歉。

> 我说诉说的自己为mod文件的lang里面加入汉化文件，每次启动游戏之后都得卸载I18n资源包才会汉化生效的事情，你们自己都不在乎。

未能成功复现，见此：

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/assets/37480168/0151eaa1-f355-44f6-abd4-30bfd2b074c0

> （对于动物名称的称呼，认为没有必要过分严谨，只要不叫错拉丁文学名即可）（你们不应该就别名提出翻译建议）

关于这一点，我与你的观点无法达成一致。我们自始至终只是提出“建议”，该词可见于我和 Chuijk 的评论的表头。"能用的译名千千万万个，但好用的译名只有一个"，我们提供学术名称，是为了给双方更多的译名选择，择优应用，从而提高整体的译名水准。另外，我们确实习惯上不论原文是否写的俗称，都一律译为中文学名。这一做法的正确与否我们没有定义，都是可以被社区讨论推翻的。我也注意到你倾向遵循原文用俗称，译文就用俗称的原则，这一做法我是肯定的，因为只要单个模组（或及其联动模组）中的翻译法则相同就可以——我没有在之前的评论中强调这一点，算是我的疏忽。

我们尊重每一位社区成员的观点。但是还请理性发言和做事。在各来源查询物种信息费时费力，你可以不接受建议，但还请尊重我们的劳动。衷心希望你在日后的社区活动中遇到其他成员的反对意见，能够做到或合理反驳，或拒绝修改，或接受建议，而不是拍屁股走人。

**Pull request closed without merging**